lib/common.js

```diff
@@ -430,7 +430,7 @@ exports.createTableForProperties = function(type, opts, properties) {
 
 exports.createTableForTeams = function(teams) {
   var table = new Table({
-    head: ['id', 'Name'],
+    head: ['ID', 'Name'],
```

Comment:
OKs. I will revert. This change was just to follow the same way that it is another places. ( the ID label)